### ISSUE TITLE
Maintain version.minor tags

### ISSUE THREAD

**Issue by vovimayhem** [google-github-actions/deploy-cloudrun#317]
### TL;DR

Create & maintain minor release tags such as `v0.8`, `v0.9`, etc. to prevent breaking changes from breaking deployments when pinned to the major release, which is the only tag available.

### Detailed design

While pinning to the latest major version works (There's a [`v0` tag](https://github.com/google-github-actions/deploy-cloudrun/tree/v0)), the same is not available for each minor version - No `v0.8` tag.

Given that newer minor version releases might contain breaking changes - such as [the `v0.9.0` release](https://github.com/google-github-actions/deploy-cloudrun/releases/tag/v0.9.0), pinning to the major version will cause broken deploys at some point.

This is a request to maintain minor release tags such as ` v0.8`, `v0.9`, so that we get bug fixes and other non-breaking changes, without the risk of our build breaking due to updates.

**Comment by sethvargo** [comment]
Hi @vovimayhem 

Thank you for opening an issue. We have a note in the README of some of the other actions, but we must have missed this one. I opened https://github.com/google-github-actions/deploy-cloudrun/pull/318 to add it.

In short, we're only going to maintain `major` and `major.minor.patch` versions. Most actions, including GitHub-owned ones, don't follow semantic versioning because it doesn't really make sense for these kinds of projects.

If you have really specific concerns, I'd recommend pinning to a specific SHA, since all Git tags are ultimately immutable.

**Comment by vovimayhem** [comment]
I'm seeing the suggested changes in the PR... and i think following the suggestion in the read me WILL CONTINUE BREAKING builds...

I'd suggest dropping the suggested edits, and instead put what you have just suggested to me in this Issue instead.
